app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1 class="text-2xl text-center mb-3">Public Recipes</h1>
+
+<div class="flex flex-col w-[40%] mx-auto">
+  <% @public_recipes.each do |recipe| %>
+    <%= render recipe %>
+  <% end %>
+</div>
```

Comment:
- [ ] Kindly make your public recipes match the wireframe design and contain the same information

| Your design  | Wireframe design|
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/53402600/208106155-8e33fa83-3136-413e-a538-d90be5b76886.png)  | ![image](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/public_recipe_index.png?token=GHSAT0AAAAAAB2ZFXSLSIHOSUWYO5TWK5EQY44PPBA) |

Reply:
I will make the changes thank you

Reply:
Changes made and fixed